device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -376,6 +376,11 @@ private void commonConstructorVerifications() throws UnsupportedOperationExcepti
      */
     public void open() throws IOException
     {
+        if (this.ioTHubConnectionType == IoTHubConnectionType.USE_MULTIPLEXING_CLIENT)
+        {
+            throw new UnsupportedOperationException("Cannot open a multiplexed client through this method. Must use multiplexingClient.registerDeviceClient(deviceClient)");
```

Comment:
Might as well make this string a constant instead of repeating it.

Reply:
I don't see this string being used in more than one place

Reply:
I see. It looks like the repeated string is the one for close.